lib/pages/invite-people-page.js

```diff
@@ -63,4 +65,9 @@ export default class InvitePeoplePage extends BaseContainer {
 
 		return DriverHelper.isElementPresent( self.driver, self.noticeSelector );
 	}
+
+	backToPeopleMenu(){
```

Comment:
I added a couple of lines more to the method so it does not break when attempting to return to the sidebar menu on mobiles. I usually store all selectors or specific methods that are unique for a certain page in a page object, just also to improve readability in the spec. Am I not seeing a potential downside?

Reply:
With this change it's working for me on mobile as well. 👍 

I don't think this method is really necessary, since we have a set of components (along with page objects) that we tend to use directly in the specs. Those components (like the top navbar and the sidebar) and their selectors/methods aren't specific to certain pages. I prefer using them directly (instead of wrapping them in a page-specific method) because it provides clarity in the spec about whether you're interacting with a page-specific element or another component, but this approach also works.

One example of a spec using the sidebar component directly:
https://github.com/Automattic/wp-e2e-tests/blob/master/specs/wp-view-site-spec.js